lib/objects/objectsInMemFileDB.js

```diff
@@ -98,6 +101,27 @@ class ObjectsInMemoryFileDB extends InMemoryFileDB {
         }
     }
 
+    /**
+     * Checks if given Id is a meta object, else throws error
+     *
+     * @param {string} id to check
+     * @throws Error if id is invalid
+     */
+    async validateMetaObject(id) {
+        if (this.metaCache[id] === undefined) {
+            // if not cached -> getObject
+            const obj = await this.getObjectAsync(id);
+            if (obj && obj.type === 'meta') {
+                this.metaCache[id] = true;
+            } else {
+                this.metaCache[id] = false;
```

Comment:
Do we really need to store "false"? I personally would only store true, but in the end also works that way

Reply:
The logic here is: 
- If true `Information cached -> and its a meta object`,
- if false `Information cached -> and its not a meta object, so no need to use getObject, we already know that its not ok`, 
- if undefined: ` not cached yet` try to get information.

Reply:
kk